mLRS/CommonTx/cli.h

```diff
@@ -710,10 +749,10 @@ bool rx_param_changed;
         if (is_cmd("help"))  { print_help(); } else
         if (is_cmd("?"))     { print_help(); } else
         if (is_cmd("v"))     { print_device_version(); } else
-        if (is_cmd("pl"))    { print_config_id(); print_param_list(0); } else
-        if (is_cmd("pl c"))  { print_config_id(); print_param_list(1); } else
-        if (is_cmd("pl tx")) { print_config_id(); print_param_list(2); } else
-        if (is_cmd("pl rx")) { print_config_id(); print_param_list(3);
+        if (is_cmd("pl"))    { print_config_id(); output_flag = 0; print_param_list(); } else
```

Comment:
that's a bit weired from a logic point of view, as the output state machine is setup in mixed places, not so nice architecture wise. I guess I would separate into two functions, one to start the print and one to handle the prints. If one keeps on with the multiple fields
if all fields can be condensed into on enum, one would need just one handler, and could set the task here

for the print_help() we have the same logocal issue that the same function needs to do both start/trigger and handling. There it's even weirder as the output_index is used to start the state machine.

I'm not sure form scratch what the nicest solution would be but I think we could spend few efforts on making this look nice

Reply:
Is the latest commit better?  It now follows the pattern you had established with CLI_STATE_STATS